MAUI/TreeMap/migration.md

```diff
@@ -0,0 +1,1450 @@
+<td>RangeColorValuePath</td>
+<td>Gets or sets the RangeColorValuePath property. This property is used to set the value path based on color value.</td>
+</tr>
+
+<tr>
+<td>{{'[HightlightColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.SfTreeMap.html#Syncfusion_SfTreeMap_XForms_SfTreeMap_HightlightColor)'| markdownify }}</td>
+<td>SelectedItemStroke</td>
+<td>Gets or sets the Selected Item Stroke property. This property is used to set the color to highlight the items.</td>
+</tr>
+
+<tr>
+<td>{{'[HightlightBorderWidth](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.SfTreeMap.html#Syncfusion_SfTreeMap_XForms_SfTreeMap_HightlightBorderWidth)'| markdownify }}</td>
+<td>SelectedItemStrokeWidth</td>
+<td>Gets or sets the Selected ItemStrokeThickness property. This property is used to customize the width of highlighted stroke.</td>
+</tr>
+
+<tr>
+<td>{{'[LayoutType](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.SfTreeMap.html#Syncfusion_SfTreeMap_XForms_SfTreeMap_LayoutType)'| markdownify }}</td>
+<td>LayoutType</td>
+<td>Gets or sets the layoutType property. This property is used to customize the layout for tree map.</td>
+</tr>
+
+<tr>
+<td>{{'[ItemTemplate](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.SfTreeMap.html#Syncfusion_SfTreeMap_XForms_SfTreeMap_ItemTemplate)'| markdownify }}</td>
+<td>LeafItemTemplate</td>
+<td>Gets or sets the LeafItemtemplate property. This property is used to set data template.</td>
+</tr>
+
+<tr>
+<td>{{'[LeafItemSettings](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.SfTreeMap.html#Syncfusion_SfTreeMap_XForms_SfTreeMap_LeafItemSettings)'| markdownify }}</td>
+<td>LeafItemSettings</td>
+<td>Gets or sets the LeafItemSettings property. This property is used to customize the settings for leaf level.</td>
+</tr>
+
+<tr>
+<td>{{'[LeafItemColorMapping](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.SfTreeMap.html#Syncfusion_SfTreeMap_XForms_SfTreeMap_LeafItemColorMapping)'| markdownify }}</td>
+<td>LeafItemBrushSettings</td>
+<td>Gets or sets the LeafItemColorSettings property. This property is used to set color settings for leaf items.</td>
+</tr>
+
+<tr>
+<td>{{'[LegendSettings](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.SfTreeMap.html#Syncfusion_SfTreeMap_XForms_SfTreeMap_LegendSettings)'| markdownify }}</td>
+<td>LegendSettings</td>
+<td>Gets or sets the Legendsettings property. This property is used to customize the settings for legend.</td>
+</tr>
+
+<tr>
+<td>{{'[Levels](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfTreeMap.XForms.SfTreeMap.html#Syncfusion_SfTreeMap_XForms_SfTreeMap_Levels)'| markdownify }}</td>
+<td>Levels</td>
+<td>Gets or sets the levels property. This property is an list collection of TreeMapleve.</td>
+</tr>
```

Comment:
Spelling mistake in tree map level.

Gets or sets the levels property. This property is an list collection of TreeMaplevef.

Reply:
Content changed